mat64/matrix.go

```diff
@@ -409,7 +409,7 @@ func Dot(a, b Matrix) float64 {
 	return sum
 }
 
-// Equal returns whether element a and b have the same size and contain all equal
+// Equal returns whether the matrices a and b have the same size and contain all equal
```

Comment:
The wording of this is weird. An interpretation could be that all elements of a and b must be the same.